bundles/tools.vitruv.applications.pcmumlcomponents/src/tools/vitruv/applications/pcmumlcomponents/uml2pcm/UmlToPcm.reactions

```diff
@@ -319,8 +319,9 @@ routine addOperationParameter(uml::Operation umlOperation, uml::Parameter umlPar
 	action {
 		call {
 			if (umlParameter.direction == ParameterDirectionKind.RETURN_LITERAL) {
-				val returnParameters = umlOperation.ownedParameters.filter[p | p.direction == ParameterDirectionKind.RETURN_LITERAL]
-				if (returnParameters.length > 0) {
+				if (umlOperation.ownedParameters.exists [ 
+					direction == ParameterDirectionKind.RETURN_LITERAL && it != umlParameter
+				]) {
```

Comment:
This was a bug that was found because the new `TestUserInteraction` throws an exception for unexpected notifications.